### ISSUE TITLE
CS to support the ability to completely initialize a venue via the Management Console

### ISSUE THREAD

**Issue by LucaCinquini** [unity-sds/unity-sps#69]
Description: In order to deploy SPS to the EMIT venue, we need the venue to completely initialized by the Common Services infrastructure. Galen suggested we should rely on the CS Management Console to be mature enough by Sprint 2 to be able to do so.

Acceptance Criteria: 
o The ESK+SPS deployment succeeds after the venue infrastructure has been created through the Management Consols

**Comment by galenatjpl** [comment]
@LucaCinquini @jpl-btlunsfo I think this ticket might be a duplicate of https://github.com/unity-sds/unity-cs/issues/351, or at least they overlap heavily. It's probably okay to keep both of these, I'm just letting you know.

**Comment by LucaCinquini** [comment]
It probably is, but I agree, let's keep them both. Thanks.

**Comment by mike-gangl** [comment]
I think this is an exercise of the venue setup guides. See how it goes and where the gaps in documentation are.

**Comment by GodwinShen** [comment]
Blocked by creation of new EMIT venue in MCP, https://github.com/unity-sds/unity-project-management/issues/151

Related to https://github.com/unity-sds/unity-data-services/issues/352
